CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@
 
 * Base any changes on the latest dev branch (e.g., `v4.0/dev`).
 * Create a topic branch for each new contribution.
-* Fix only one problem at a time. This helps to quickly test and merge submitted changes. If intending to fix *multiple unrelated problems* then use a separate branch for each problem.
+* Fix only one problem at a time. This helps to quickly test and merge submitted changes. If you intend to fix *multiple unrelated problems*, please use a separate branch for each problem.
```

Comment:
The documentation is written in the style of a reference document and generally avoids the use pronouns, i.e. instead of something like "You need to copy the rule files into place" use "The rule files need to be copied into place". Similarly, I avoid using unnecessary pleasantries like "please do this" etc., which works fine in our blog posts and conversational pieces but the tone here is more of a reference guide.

Reply:
Fixed in [654d1ba](https://github.com/coreruleset/coreruleset/pull/3292/commits/654d1baa114efd62c7b8c250301a838c03794236). Also added the comment into doc 👍